homeassistant/components/growatt_server/sensor.py

```diff
@@ -554,6 +555,7 @@
         vol.Optional(CONF_PLANT_ID, default=DEFAULT_PLANT_ID): cv.string,
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
+        vol.Required(CONF_URL, default=DEFAULT_URL): cv.string,
```

Comment:
We should not extend the platform schema with a new option. The platform config is legacy.

Reply:
#54325